### DISCUSSION TITLE
Anyone willing to share their corfu configuration?

### DISCUSSION THREAD

**Discussion by kartikynwa** [meow-edit/meow#702] [General]
I am trying to set up corfu for completion. I would prefer a configuration that enables corfu-auto.

The problem I am facing is that I see some weird behaviour from corfu and I don't know if the interaction with meow mode has anything to do with it. For example, sometimes the corfu popup wom't close. As such if someone can share their configuration I would use it as a starting rather than trying to figure out everything from scratch.

Thanks.

**Comment by jgarte** [comment]
Here's my corfu config:

https://toys.whereis.social/symbols?search=feature-emacs-corfu

**Reply by kartikynwa** [reply]
Thank you
